config/routes.rb

```diff
@@ -8,4 +8,5 @@
   root "home#index"
   
   get "/health_check", to: 'health_check#health_check', as: :rails_health_check
+  get "/search_stats", to: 'search_stats#index'
```

Comment:
Please take a look at [Rails Routing](https://guides.rubyonrails.org/routing.html#crud-verbs-and-actions). We can keep the current implementation, but it should better follow the Rails Routing as we have `search_stats` as a persisted resource.

```suggestion
  resources :search_stats, only: [:index]
```

Reply:
Good idea. It would be better to use the suggested one.